octoprint_mrbeam/__init__.py

```diff
@@ -855,6 +855,7 @@ def on_ui_render(self, now, request, render_kwargs):
                     self.get_model_id()),
                 burger_menu_model=BurgerMenuService(self._logger, DocumentService(self._logger)).get_burger_menu_model(
                     self.get_model_id()),
+                isDevelop=self.is_dev_env(),
```

Comment:
can this easily be changed by the user? So you 'hack' your device to be a develop one

Reply:
I am not sure about this... But I did not change that behaviour, I am only explicitly exposing it to the jinja2 templating engine...